src/components/Header/Header.jsx

```diff
@@ -1,18 +1,20 @@
 import React from 'react';
 import { Button, Typography, Box } from '@material-ui/core';
+import { useStyles } from './style';
 
 export default function Header() {
+  const classes = useStyles();
   return (
     <Box className="header">
-      <Typography className="joinUs">Join Us</Typography>
-      <Typography variant="h2" className="headerHeader">
-        COURSES OF EXCELLENT QUALITY
+      <Typography className={classes.joinUs}>Join Us</Typography>
+      <Typography variant="h2" className={classes.header}>
+        Courses of excellent quality
       </Typography>
-      <Typography variant="h5" className="headerText">We know how large objects will act, but things on a small scale just do not act that way</Typography>
-      <Button variant="contained" color="primary">
+      <Typography variant="h5" className={classes.headerDescription}>We know how large objects will act, but things on a small scale just do not act that way</Typography>
+      <Button className={classes.buttonQuote} variant="contained">
         Get Quote Now
       </Button>
-      <Button variant="outlined" color="primary">
+      <Button className={classes.buttonLearn} variant="outlined">
```

Comment:
Much better, you may leave as it is now - if there will be some of them but with different texts, you may want to try ```<button className={classes.ctaButton} text={buttonText} />``` but now seems legit